app/assets/stylesheets/mobile/header.css.scss

```diff
@@ -169,10 +163,20 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
-       -o-transform: translateX(-80%);
-      -ms-transform: translateX(-80%);
-     -moz-transform: translateX(-80%);
-  -webkit-transform: translateX(-80%);
+.draw {
```

Comment:
Did you test this on every browsers? I remember I had to use this > * to deal with a "bug" in the CSS interpretor which does not react correctly when applying a translate on an element with `position: fixed;` on it. (See http://stackoverflow.com/questions/9364203/position-fixed-div-is-not-fixed-when-parent-rotates-or-translates)

Here you're translating the whole application so I suppose this works (and we will not have the weird reaction of the stream translating at the end of the closing of the drawer) but I want to be sure you tested it: I had a lot of trouble with the menu not opening on iPad, Opera Mini...